packages/relative-values/src/components/View/ListView/index.tsx

```diff
@@ -73,6 +60,161 @@ export const ListView: FC<Props> = ({ model }) => {
     </CellBox>
   );
 
+  return (
+    <div className="mb-10">
+      <div className="">
+        <div
+          className="grid border-r border-b border-gray-200 w-full"
+          style={{
+            gridTemplateColumns: showDescriptions ? "2fr 2fr 1fr" : "3fr 1fr",
+          }}
+        >
+          {headerRow("Name")}
+          {showDescriptions && headerRow("Description")}
+          <ColumnHeader
+            selectedItem={denominatorItem}
+            setSelectedItem={setDenominatorItem}
+          />
+          {sortedItems.map((item) => (
+            <Fragment key={item.id}>
+              <CellBox>
+                <div
+                  className="p-2 flex justify-end text-slate-700 cursor-pointer font-bold text-sm"
+                  onClick={() => setDenominatorItem(item)}
+                >
+                  <div className="flex-grow text-sm hover:text-slate-900 hover:underline">
+                    {item.name}
+                  </div>
+                  <div
+                    onClick={() => setDenominatorItem(item)}
+                    className={"pl-2 cursor-pointer"}
+                  >
+                    <CompassIcon
+                      className={clsx(
+                        "hover:fill-slate-800",
+                        item.id === denominatorItem.id
+                          ? "fill-slate-500"
+                          : "fill-slate-200"
+                      )}
+                      size={23}
+                      viewBox={"0 0 25 25"}
+                    />
+                  </div>
+                </div>
+
+                <div className="flex px-2 pb-3">
+                  {item.clusterId !== undefined ? (
+                    <div className="flex gap-1 items-center">
+                      <div className="flex-none opacity-50">
+                        <ClusterIcon
+                          cluster={catalog.clusters[item.clusterId]}
+                          selected={true}
+                        />
+                      </div>
+                      <div className="text-xs font-medium text-slate-400">
+                        {catalog.clusters[item.clusterId].name}
+                      </div>
+                    </div>
+                  ) : (
+                    ""
+                  )}
+                  <div className="px-3 font-mono text-xs text-slate-400">
+                    {item.id}
+                  </div>
+                </div>
+              </CellBox>
+              {showDescriptions && (
+                <CellBox>
+                  <div className="p-3 text-sm text-slate-500">
+                    {item.description}
+                  </div>
+                </CellBox>
+              )}
+              <div
+                className="cursor-pointer"
+                onClick={() => {
+                  if (!!sidebarItems && sidebarItems[0].id === item.id) {
+                    setSidebarItems(undefined);
```

Comment:
After playing with it, "Click to deselect" feels awkward to me, especially since the currently selected cell is not highlighted. Too easy to click the same cell again by accident, and then the the table moves and I have to point to a different position on screen to bring it back.

The close "x" button on the sidebar might be better.

Reply:
I agree that closing would be better, will likely do that later. This is kind of hacky, but okay doing this in increments.